analytics_data_api/v0/views/courses.py

```diff
@@ -810,3 +814,52 @@ def get(self, _request, course_id, report_name):
             return Response(response)
         else:
             raise ReportFileNotFoundError(course_id=course_id, report_name=report_name)
+
+
+# pylint: disable=abstract-method
```

Comment:
I was confused by this, so I pulled to check whether it was necessary. It looks like `make quality` runs cleanly without this line; could you remove it?